manifests/base/influxdb/influx-config.yaml

```diff
@@ -2,7 +2,6 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   name: influx-config
-  namespace: b4mad-racing-test
```

Comment:
that will be added again by running
```
helm template influxdb2 bitnami/influxdb \
  --values influxdb2-values.yaml \
  --output-dir influxdb2
```

in the future. 🤷

Reply:
ja, so re need to remove it again in the future :P